### PR TITLE
Improve Ukrainian localization

### PR DESCRIPTION
**Pull request by sftim**
Rolling up some changes made by @andygol

### CODE REVIEW

**Comment by sftim** [comment]
CLA check is passing; @Andygol can you confirm these commits are all your own work?

:information_source: you're also welcome to open a pull request directly.

**Comment by Andygol** [comment]
> CLA check is passing; @Andygol can you confirm these commits are all your own work?

Yes. All this is my own work. You may find some of my announcements on LinkedIn regarding this (See https://www.linkedin.com/posts/andrii-holovin-b58201122_робота-над-перекладом-документації-kubernetes-activity-7166731616696967168-jL0Y/?utm_source=share&utm_medium=member_desktop).

**Comment by sftim** [comment]
I worry that these changes might be too large to review; however, I'll let reviewers for this localization decide.

**Comment by sftim** [comment]
@Andygol, I could split this out into four or five smaller PRs squashed down from the commits on this branch. If you're happy to do that, I'll open the PRs.

(because this is open source, anyone else is also welcome to do that)

**Review by sftim — COMMENTED** [review]
/hold

This PR includes untranslated English content.

**Comment by Arhell** [comment]
> I worry that these changes might be too large to review; however, I'll let reviewers for this localization decide.

Thank you, you are right, don't need to big of PR
It’s better to have a lot and not big ones, so that you can check and merge with the main branch.
It's my opinion

**Comment by Andygol** [comment]
> I could split this out into four or five smaller PRs

👍  It's fine. 

In case we were doing similar work on some localization platform, the workflow would be much easier 😉

**Comment by MaxymVlasov** [comment]
How the hell it should be reviewed in GH...

File: content/uk/case-studies/appdirect/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - Company: AppDirect
+  - Location: San Francisco, California
+  - Industry: Software
+---
+
+<h2>Challenge</h2>
+
+<p><a href="https://www.appdirect.com/">AppDirect</a> provides an end-to-end commerce platform for cloud-based products and services. When Director of Software Development Pierre-Alexandre Lacerte began working there in 2014, the company had a monolith application deployed on a "tomcat infrastructure, and the whole release process was complex for what it should be," he says. "There were a lot of manual steps involved, with one engineer building a feature, then another team picking up the change. So you had bottlenecks in the pipeline to ship a feature to production." At the same time, the engineering team was growing, and the company realized it needed a better infrastructure to both support that growth and increase velocity.</p>
+
+<h2>Solution</h2>
```
Comment (MaxymVlasov) [inline content/uk/case-studies/appdirect/index.html:27]: Hmm. basically, there not translated 90%

**Comment by natalisucks** [comment]
`I propose blindly approving that and hope it does not contain some non-optimal stuff inside`

@MaxymVlasov Please do not do this. As already suggested in prior comments, this PR is too large to review and needs to be broken down into smaller PRs so that reviewers can have an easier job to do. "Blind approving", and suggesting to do so, undermines the trust that the Kubernetes project, and our SIG, have put into you as a reviewer and approver of this language.

**Comment by Andygol** [comment]
@natalisucks, what is your thought about give it a fly as a pilot project, using one of the centralized translation management platforms to streamline the process?

re Discussions: https://github.com/kubernetes/website/discussions/45209, https://github.com/kubernetes/website/discussions/45438

**Comment by natalisucks** [comment]
@Andygol per greater discussion with the UK team in #45175, the next step regarding possible adoption of a translation platform, as I understood, was formulating a team to try and push a proof-of-concept for using this kind of platform, in the hope that several localization teams could weigh in and comment/help

**Comment by Andygol** [comment]
@natalisucks I hope that in these efforts, SIG Docs will not stand aside and only observe, but will also take an active part in this project as its owner. 👭👬👫🧑‍🤝‍🧑🤝 🧑‍💼
I hope for fruitful and mutually beneficial cooperation on this project between the localization teams and SIG Docs.

**Comment by sftim** [comment]
> @natalisucks I hope that in these efforts, SIG Docs will not stand aside and only observe, but will also take an active part in this project as its owner. 

Hi. We're low on capacity for new initiatives, but that's not the same as _zero_ capacity. I hope we can find folk to staff the efforts around experimenting with localization automation.
In the meantime, please do follow our norms around making changes to the live site; if you're stuck because of team capacity, let's surface that and work on how to recruit more team members who speak Ukrainian.

BTW: _this_ PR probably should not merge. See https://github.com/kubernetes/website/pull/45371#issuecomment-1967604208

**Comment by natalisucks** [comment]
My recommendation is to close this PR, unless the draft is still being used as a reference to break this down into smaller pieces of work

**Comment by sftim** [comment]
> https://github.com/kubernetes/website/pull/45371#issuecomment-2090044621

Exactly that. There's such a big gap between _main_ and the fork these are from that it will need a lot of work to bring in the changes, and this serves to remind me that it needs doing.

**Pull request closed without merging**